### ISSUE TITLE
Contrast - global

### ISSUE THREAD

**Issue by vavroom** [nicolas-steenhout/main#87]
◦	Global - Might be fixed with your upcoming style edits, but in Safari, there’s very poor contrast in light mode and some poor contrast in dark mode. I think some of the backgrounds haven’t loaded? Tried on multiple devices. Issues within the header and footer, link contrast, etc. Could be a problem with inheritance in the CSS?

**Comment by digilou** [comment]
@vavroom Is this still an issue? If so, do you have a screenshot for this? Is it just Safari? What version?

**Comment by vavroom** [comment]
Cannot replicate. Did not get any screenshots from Alicia. Closing as "can't fix" because I don't think it's an issue.

**Comment by digilou** [comment]
@vavroom It _may_ be an issue. What I'm wondering is Safari versioning, and if I'll need a fallback for `light-dark`. It's pretty new, but does work for all the newest versions of major browsers. Which is why I was so excited about it. Definitely won't work for IE6... ;)
